hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -107,28 +107,44 @@
     {
       name: pattgen_op_cg
       desc: '''
-            The following coverage strategy is implemented:
-            * The counters are covered at the max
-              and min corner cases and for the middle values
-            * Data bus has functional coverage using walking ones and zeroes
-            * One bit width variables are wrapped in one cover point
-              while counters and data have designated cover points
-            * Channel enable signal is stand alone cover point
+            Covers the interrupt done and event done for channel 0 and 1.
             '''
     }
     {
       name: roll_cg
       desc: '''
-            Covers that all the counters revert to zero when they reach
-            the maximum value.
-            Individual initial and maximum counter values
+            Covers the individual initial and maximum counter values
             that will be covered include:
             - Initial and maximum values of clock divide counter
             - Initial and maximum values of length counter
             - Initial and maxim values of repetition counter
             All valid combinations of the above will also be crossed.
             '''
     }
+    {
+      name: pattgen_ch0_cnt_reset_cg
+      desc: '''
+            Covers that ch0 counter values resets to zero when it
+            reaches the assigned value.
```

Comment:
I believe this hjson file follows 100 character limit. Maybe you can help update the first line to reach the limit?